core/gui/src/app/workspace/component/code-editor-dialog/code-editor.component.ts

```diff
@@ -296,30 +402,23 @@ export class CodeEditorComponent implements AfterViewInit, SafeStyle, OnDestroy
     this.showAnnotationSuggestion = false;
     this.currentCode = "";
     this.currentSuggestion = "";
+
+    // Only for "Add All Type Annotation"
+    if (this.isMultipleVariables && this.userResponseSubject) {
+      this.userResponseSubject.next();
+    }
   }
 
-  // Add the type annotation into monaco editor
   private insertTypeAnnotations(
```

Comment:
are you inserting one annotation or multiple annotations in this method?

Reply:
Only insert one annotation.